src/components/snapshot/overview/SnapshotOverviewPanel.tsx

```diff
@@ -150,6 +151,33 @@ function SnapshotOverviewPanel(props: SnapshotOverviewPanelProps) {
             <Typography variant="h6">Storage:</Typography>
             {renderStorageResources(sourceDataset)}
           </Grid>
+          {sourceDataset.phsId && (
+            <Grid item xs={4}>
+              <Typography variant="h6">PHS ID:</Typography>
+              {sourceDataset.phsId}
+            </Grid>
+          )}
+          <Grid item xs={4}>
+            <EditableFieldView
+              fieldValue={snapshot.consentCode}
+              fieldName="Consent Code"
+              canEdit={isSteward}
+              updateFieldValueFn={(text: string | undefined) => {
+                dispatch(patchSnapshot(snapshot.id, { consentCode: text }));
+              }}
+              useMarkdown={false}
+            />
+          </Grid>
+          <Grid item xs={4}>
+            <Typography variant="h6">Billing Profile Id:</Typography>
+            {snapshot.profileId}
+          </Grid>
+          {snapshot.dataProject && (
+            <Grid item xs={4}>
+              <Typography variant="h6">Google Data Project:</Typography>
+              {snapshot.dataProject}
+            </Grid>
+          )}
         </Grid>
```

Comment:
I'm not sure exactly where this is specified, and I don't think you added it, but in this [example snapshot](https://jade-sh.datarepo-dev.broadinstitute.org/snapshots/fe7f9842-3f70-4027-b536-648b7807900a) I also see "Creation Information:" below these fields, with nothing in it.  Can this be rendered conditionally or otherwise populated?

![Screen Shot 2022-11-03 at 11 59 14 AM](https://user-images.githubusercontent.com/79769153/199771332-56b1d35f-23d2-41e5-91ed-2040384e22df.png)